department-of-veterans-affairs/va.gov-cms
# VACMS-6007: Add banner content type.

## Description
Closes #6007 

## Testing done
Visual / behat

## Screenshots
![image](https://user-images.githubusercontent.com/2404547/127905712-682b8730-6e09-4eff-9c59-04ed9676d3a1.png)

## QA steps

As user _uid_ with _user_role_
As a content_admin, go to `admin/structure/types/manage/banner/form-display` an visually verify banner type exists.

### Select Team for PR review


### Is this PR blocked by another PR?


### Does this PR need review from a Product Owner


### CMS user-facing annoucement

Is an announcement needed to let editors know of this change?
